docs/develop/scripting-fundamentals.md

```diff
@@ -316,6 +316,7 @@ The [Office Scripts API reference documentation](/javascript/api/office-scripts/
 
 ## See also
 
+- [Office Scripts Dev Center](https://developer.microsoft.com/office-scripts)
```

Comment:
Can we remove this link? The customer is in a basics article and I think that they should go to more technical content, rather than a hub page.

Reply:
Moved to end of list.